### DISCUSSION TITLE
Problemas de deslogearse - LOGOUT Issues

### DISCUSSION THREAD

**Discussion by josecede01** [chamilo/chamilo-lms#5842] [General]
hola estimados..

acabo de instalar chamilo ultima version en mi VPS de godaddy  desde el instalador de paquetes CPANEL...  todo bien al momento de la instalacion..

el problema real es que una vez entro con las credenciales de Administrador  y al finalizar realizo el LOGOUT  pero este en vez de ir a la pantalla de inicio para colocar las credenciales se queda en la pagina del usuario, y al intentar entrar al menu administracion donde me sale el mensaje de logearme  **### You are not allowed to see this page. Either your connection has expired or you are trying to access a page for which you do not have the sufficient privileges.**

![12](https://github.com/user-attachments/assets/3ea4f1de-39ba-4a30-9483-2364a7e6dc8a)
![1](https://github.com/user-attachments/assets/003832c9-7bd7-43db-9092-6043a9aaeb76)

**Comment by Oscaruix** [comment]
Me pasa lo mismo pero no tengo idea como arreglarlo... Te han dado alguien alguna opcion?

**Reply by josecede01** [reply]
Hola si, luego de volver e intentar instalar mil veces hable con soporte de mi hosting y el problema estaba en ellos tenia doble firewall y las reglas de seguridad.. daban este problema..  por cierto mi hosting es godaddy

**Comment by Oscaruix** [comment]
Muchas gracias
